### ISSUE TITLE
how to use the policies in onResponse

### ISSUE THREAD

**Issue by enieber** [viniciusdacal/redux-arc#11]
in the documentation have the example of code how to use policies only with beforeRequest, how use with onResponse?

**Comment by viniciusdacal** [comment]
We still need to improve the docs, to include this and other examples.

I wrote an article recently, that approaches that (and all arc's core functionalities):

[ARC -  Simplifying async requests in Redux apps](https://medium.com/front-end-hacking/arc-simplifying-async-requests-in-redux-apps-e8052b874216)
[ARC - Simplificando requisições assíncronas em Aplicações Redux](https://medium.com/@viniciusdacal/arc-simplificando-requisi%C3%A7%C3%B5es-ass%C3%ADncronas-em-aplica%C3%A7%C3%B5es-redux-b9ec17fd6af6)[pt-br]

But in summary, you only need to change the applyPoint to `onResponse`

```js
function myPolicy(store) {
  return done => (action, error, response) => {
    // if you want, you can do something with the response here.
    return done(action, error, response);
  }
}

myPolicy.applyPoint = 'onResponse';
```

**Comment by viniciusdacal** [comment]
Will let the issue opened till we write the example

**Comment by viniciusdacal** [comment]
closed by #13
